test/gtest/ucs/test_sys.cc

```diff
@@ -137,3 +184,25 @@ UCS_TEST_F(test_sys, memunits_to_str) {
     test_memunits(UCS_TBYTE, "1T");
     test_memunits(UCS_TBYTE * 1024, "1024T");
 }
+
+UCS_TEST_SKIP_COND_F(test_sys, memcpy, RUNNING_ON_VALGRIND || !ucs::perf_retry_count) {
+    const double diff      = 0.95; /* allow 5% fluctuations */
+    double memcpy_bw       = 0;
+    double memcpy_relax_bw = 0;
+    size_t size;
+    char memunits_str[256];
+    int i;
+
+    for (size = 4096; size <= 256 * UCS_MBYTE; size *= 2) {
+        ucs_memunits_to_str(size, memunits_str, sizeof(memunits_str));
+        UCS_TEST_MESSAGE << memunits_str;
+        for (i = 0; i < ucs::perf_retry_count; i++) {
+            memcpy_bw       = measure_memcpy_bandwidth<memcpy>(size);
+            memcpy_relax_bw = measure_memcpy_bandwidth<memcpy_relaxed>(size);
+            if (memcpy_relax_bw / memcpy_bw >= diff) {
```

Comment:
pls print the BW results for each retry (instead of just size in line 198)

Reply:
added info for bandwidth + number of iterations